include/spm/rel/an.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <common.h>
+#include <spm/evtmgr.h>
+#include <evt_cmd.h>
+
+CPP_WRAPPER(spm::an)
+
+EVT_DECLARE_USER_FUNC(evt_an_darkness_onoff, 1);
+EVT_DECLARE_USER_FUNC(evt_an_remove_npcs, 1);
```

Comment:
Add a comment along the lines of `// more` since this doesn't cover the full bounds of the file